ultimaker_abs_black.xml.fdm_material

```diff
@@ -19,6 +19,20 @@
         <weight>750</weight>
     </properties>
     <settings>
+        <setting key="anti ooze retract position">-4</setting>
```

Comment:
I think we only want this on the generic material? Or why is it in the specific colored materials? The settings are the same as generic_ABS

Reply:
I followed the same structure I found in other materials, where the setting is in both the generic and the Ultimaker files. See [ultimaker_pla_green](https://github.com/Ultimaker/fdm_materials/blob/master/ultimaker_pla_blue.xml.fdm_material) for example. Do UM materials "inherit" from the generic?

Reply:
I think so, but to make sure, check with Cura team (I can join if you want).

Reply:
No, UM materials do NOT inherit from generic, so it needs to be specified again.

Reply:
Good to know!